pipelines/_ingredients/base-v2.yaml

```diff
@@ -880,7 +879,11 @@ subsets:
       - makeAnalysisSourcePhotometricRefMatch
       - analyzeSourcePhotometricRefMatch
 
-  step4b-measure-variability-tracts:
+  step4b-generate-ephemerides:
```

Comment:
Need to have `-global` in the step name.

Reply:
And add a description saying that it inputs are ready after step1c (and can therefore be run in parallel with step4a).  Oh, I just realized that if it is living down here it can use the `visit_summary` instead of `preliminary_visit_summary` for its WCS's.